tests/kconfig_features/Makefile

```diff
@@ -137,6 +137,10 @@ BOARD_WHITELIST += 6lowpan-clicker \
                    ruuvitag \
                    same54-xpro \
                    samr21-xpro \
+                   saml21-xpro \
+                   samr21-xpro \
```

Comment:
This one is duplicated in the list and `saml21-xpro` should be before the already existing `samr21-xpro`

Reply:
Fixed it myself :)